target/linux/ath79/dts/ar7242_meraki_mr12.dts

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+#include "ar7242.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "meraki,mr12", "qca,ar7242";
+	model = "Meraki MR12";
+
+	aliases {
+		led-boot = &led_power_orange;
+		led-failsafe = &led_power_orange;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_orange;
+	};
+
+	extosc: ref {
+		compatible = "fixed-clock";
+		#clock-cells = <0>;
+		clock-output-names = "ref";
+		clock-frequency = <40000000>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi1 {
+			label = "green:wifi1";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi2 {
+			label = "green:wifi2";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi3 {
+			label = "green:wifi3";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi4 {
+			label = "green:wifi4";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		wan {
+			label = "green:wan";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
if we are sticking with the changes you made to the LEDs script...

```
		lan {
			label = "green:lan";
			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
		};
```

Reply:
Done.

Reply:
'wan' label is now correctly 'lan' as per [the `ucidef` in `01_leds`](https://github.com/openwrt/openwrt/pull/3634/commits/e783ef483365f08712c29411b2a73891e5cc2ae8#diff-0cb1ee09def7e126c395869276d9c81a57427535a63784b1759139368656d311R227). Marking as resolved.